package.json

```diff
@@ -8,15 +8,22 @@
     "build": "npm run clean && webpack",
     "start": "styleguidist server",
     "styleguide": "styleguidist build",
-    "test": "npm run eslint:src && jest --coverage",
+    "test": "npm run eslint:src && jest --coverage && npm run size",
     "eslint:src": "eslint ./src ./examples ./*.js",
     "commitmsg": "commitlint -e",
     "precommit": "lint-staged",
     "prepublish": "NODE_ENV=production npm run build && jest",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "logo": "cd logo && sketchtool export artboards logo.sketch",
-    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo"
+    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo",
+    "size": "size-limit"
```

Comment:
Should we add `size:why` with the `why` mode?

Reply:
Good idea!